build/migration/postgres/4_current-migrate.sh

```diff
@@ -50,33 +50,33 @@ done
 echo ">>> Persistence >>> current is being run"
 
 echo "- Update Schema"
-java -jar ${WORKSPACE}/fhir/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
+java -jar ${WORKSPACE}/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
     --db-type postgresql --prop db.host=localhost --prop db.port=5432 --prop db.database=fhirdb \
     --prop user=fhiradmin --prop password=change-password \
     --schema-name FHIRDATA --update-schema --pool-size 1
 
 echo "- Grants to FHIRSERVER"
-java -jar ${WORKSPACE}/fhir/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
+java -jar ${WORKSPACE}/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
     --db-type postgresql --prop db.host=localhost --prop db.port=5432 --prop db.database=fhirdb \
     --prop user=fhiradmin --prop password=change-password \
     --schema-name FHIRDATA --grant-to FHIRSERVER --pool-size 1
 
 echo ">>> Set up the derby databases for multidatastore scenarios"
-DB_LOC="${WORKSPACE}/fhir/build/migration/postgres/workarea/volumes/dist/derby"
-rm -rf ${WORKSPACE}/fhir/build/migration/postgres/workarea/volumes/dist/derby
+DB_LOC="${WORKSPACE}/build/migration/postgres/workarea/volumes/dist/derby"
+rm -rf ${WORKSPACE}/build/migration/postgres/workarea/volumes/dist/derby
 mkdir -p ${DB_LOC}
-java -jar ${WORKSPACE}/fhir/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
+java -jar ${WORKSPACE}/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
     --db-type derby --prop db.database=${DB_LOC}/fhirDB --prop db.create=Y \
     --update-schema
-java -jar ${WORKSPACE}/fhir/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
+java -jar ${WORKSPACE}/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
     --db-type derby --prop db.database=${DB_LOC}/profile --prop db.create=Y \
     --prop resourceTypes=Patient,Group,Practitioner,PractitionerRole,Person,RelatedPerson,Organization,Location,Observation,MedicationAdministration,StructureDefinition,ElementDefinition,CodeSystem,ValueSet,Resource \
     --update-schema
-java -jar ${WORKSPACE}/fhir/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
+java -jar ${WORKSPACE}/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
     --db-type derby --prop db.database=${DB_LOC}/reference --prop db.create=Y \
     --prop resourceTypes=Patient,Group,Practitioner,PractitionerRole,Device,Organization,Location,Medication,Observation,MedicationAdministration,StructureDefinition,ElementDefinition,CodeSystem,ValueSet,Resource \
     --update-schema
-java -jar ${WORKSPACE}/fhir/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
+java -jar ${WORKSPACE}/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
```

Comment:
I opted not to do take this approach.  It's not ideal and makes the git history polluted.  I personally prefer the two separate directories.

Reply:
I wouldn't recommend anyone do this for working with the repo, but for just checking out the code I think it'll work just fine.

Reply:
This is an anti-pattern.  While it works, I suggest altering the approach to use the sub directory.

Reply:
Would it help if we add a comment explaining what we're doing and why it is ok in this case (but not in general)?

Something like this:
```
# Note: here we are cloning a previous version of the repo into a subdirectory of itself.
# This is not advised, but in this case it is OK because it is used strictly for pulling down the code.
# Avoid this pattern for repositories that you plan to continue working with over time.
```